packages/core/src/constant.ts

```diff
@@ -32,10 +32,13 @@ export const SOLID_CLIENT_AUTHN_KEY_PREFIX = "solidClientAuthn:";
 export const PREFERRED_SIGNING_ALG = ["ES256", "RS256"];
 
 export const EVENTS = {
-  NEW_REFRESH_TOKEN: "newRefreshToken",
   ERROR: "onError",
```

Comment:
this ones a tad odd? Can we fix it to just be `"error"` ?

Reply:
Yeah I was thinking that too, and I was initially reluctant because that's technically a public API, but it's not documented anywhere (people are expected to use the `onError` callback instead), so I'll go ahead and rename this.